packages/happo/src/validateConfig.js

```diff
@@ -0,0 +1,42 @@
+function fail(message) {
+  /* eslint-disable no-console */
+  console.error('Happo config validation failed:');
+  console.error(message);
+  process.exit(1);
+}
+
+function validateConfig(config) {
```

Comment:
I added some validation code for the config. This will probably help make upgrading easier as well...